cmd/brew-services.rb

```diff
@@ -121,20 +125,39 @@ def homebrew!
       extend(FileUtils)
     end
 
-    # Access current service
-    def service; @service ||= Service.new(Formula.factory(@formula)) if @formula end
+    # Access target services
+    def target_services
+      @target_services ||= begin
+        if @formula == '--all'
```

Comment:
Please take a look at the latest change and tell me if that's something that you had in mind :)